ibm/Dockerfile.j2

```diff
@@ -33,6 +36,11 @@ RUN apt-get update -qq \
     && cd terraform-provider-null-${IBM_NULL_PROVIDER_VERSION} \
     && make build \
     && mv $GOPATH/bin/terraform-provider-null /terraform.d/plugins/terraform-provider-null_v${IBM_NULL_PROVIDER_VERSION} \
+    && wget https://github.com/terraform-providers/terraform-provider-aws/archive/v${AWS_PROVIDER_VERSION}.zip \
```

Comment:
It's becoming a messy wall of text I think. :sweat_smile: 

Can you please split the blocks by intent, and maybe add a comment / title for each block ?

Or even better : use `echo` like we did in `dind` or `java`.

Reply:
Empty lines are deprecated:
```
[WARNING]: Empty continuation lines will become errors in a future release.
```
I will split into multiple RUN statements

Reply:
Ok, split with echo

Reply:
Sure, there's not any issue with mutliple RUN statements now that the officials debian (which goland is based on through buildpack) and ubuntu are automatically cleaning after the install : https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105

At least according to the doc : https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

![image](https://user-images.githubusercontent.com/3366767/83041461-c9126b00-a040-11ea-914e-dc3b856c8ddc.png)


Before this feature, multiple RUN would render the cleaning useless, since you can't remove a file created in layer N from layer N+X.

Reply:
Oh nice, didn't know. I will split into multiple RUN statement then, give me 5min

Reply:
Done